src/types/drupal/node.ts

```diff
@@ -260,6 +261,77 @@ export interface NodeLandingPage extends DrupalNode {
   field_support_services: NodeSupportService[]
 }
 
+type InternalDrupalTarget = {
+  type: string
+  id: string
+  resourceIdObjMeta: {
+    drupal_internal__target_id: number
+  }
+}
```

Comment:
This is just an entity reference that hasn't been hydrated. Unless there's some reason for us to _use_ an unhydrated entity reference, we should either hydrate it (add it to the includes) or omit it from the type. Commenting it out is fine so we know it's technically there, but we don't actually use it.